### PR TITLE
Search user

### PR DESCRIPTION
**Pull request by silasade**
Added the "Search Users" functionality

### CODE REVIEW

File: src/Pages/Login/Index.tsx
Comment (a-k-o-r-e-d-e) [inline src/Pages/Login/Index.tsx:1]: please explain this change?

Comment (silasade) [reply src/Pages/Login/Index.tsx:1]: There is field for that in the error object if there are no status errors.

Comment (a-k-o-r-e-d-e) [reply src/Pages/Login/Index.tsx:1]: But now you no longer display the error from the backend?

Comment (a-k-o-r-e-d-e) [reply src/Pages/Login/Index.tsx:1]: And how do you know it is slow network?

Comment (silasade) [reply src/Pages/Login/Index.tsx:1]: that is the error from the backend for that particular case is not user friendly

Comment (a-k-o-r-e-d-e) [reply src/Pages/Login/Index.tsx:1]: Still display the erorr message from the backend if it exists, then you suggest to the backend to improve the error message

Comment (silasade) [reply src/Pages/Login/Index.tsx:1]: Already fixed that

**Review by a-k-o-r-e-d-e — APPROVED** [review]

**Pull request merged**